src/controller/game_controller.py

```diff
@@ -162,49 +166,23 @@ def play_one_round(self):
                             return False
 
                         elif event.key in [K_1, K_KP1]:
-                            print(1)
-                            card = self.deck.getCard()
-                            human.add_card(card, hand_idx)
-                            print("You : " + str(human.hands[hand_idx].hand))
-                            if human.hands[hand_idx].hand.is_burnt or human.hands[hand_idx].hand.is_black_jack:
-                                print("Is burnt or black jack")
-                                state = False
-
+                            self.view_game.buttons["card"].execute()
```

Comment:
Just an idea here, for later maybe, I think we should make the button looked pressed when we press the corresponding keyboard key, and call execute() only when the key is released, exactly how the do it when we press the button with the mouse pointer

Reply:
I made an issue of your idea.